htmresearch/frameworks/nlp/imbu.py

```diff
@@ -64,10 +64,13 @@ def __init__(self, exc):
 
 
 
-def _loadNetworkConfig():
+def _loadNetworkConfig(jsonName=None):
   """ Load network config by calculating path relative to this file, and load
   with htmresearch.frameworks.nlp.model_factory.getNetworkConfig()
   """
+  if jsonName is None:
```

Comment:
`if not jsonName` would catch `None` as well as empty string or anything the resolves to `False`